semgrep/Pipfile

```diff
@@ -10,9 +10,11 @@ tox = "*"
 pytest-snapshot = "*"
 appdirs = "*"
 pytest-xdist = "*"
+xmldiff = "*"
 
 [packages]
 semgrep = {editable = true,path = "."}
+junit-xml = "*"
```

Comment:
No need to put this here since the dependencies in setup.py are installed when semgrep is installed
```suggestion
```